go.mod

```diff
@@ -1,12 +1,9 @@
 module github.com/celestiaorg/celestia-app
 
-go 1.21
-
-toolchain go1.21.0
+go 1.21.1
 
 require (
-	github.com/celestiaorg/nmt v0.19.0
-	github.com/celestiaorg/quantum-gravity-bridge v2.1.0+incompatible
+	github.com/celestiaorg/nmt v0.20.0
```

Comment:
note this also bumps the NMT version

Reply:
good catch, once the go version bump is merged, I will adapt this PR, thanks

Reply:
[18a6add](https://github.com/celestiaorg/celestia-app/pull/2545/commits/18a6adda7940d679355188caba2c736f22b51248)